src/popup/router/components/TransferReview.vue

```diff
@@ -279,18 +274,20 @@ export default {
         this.openCallbackOrGoHome(true);
       } catch (e) {
         this.openCallbackOrGoHome(false);
-        await this.$store.dispatch('modals/open', {
-          name: 'default',
-          title: this.$t('modals.transaction-failed.msg'),
-          icon: 'critical',
-        });
+        this.openTransactionFailedModal();
         e.payload = { url: recipient };
         throw e;
       } finally {
         this.loading = false;
+        this.$emit('success');
```

Comment:
should we emit "success" even if the error was thrown?

Reply:
You have right, it should not. Fixed...